src/training_raster_clipper/implementation/rskandrani.py

```diff
@@ -33,9 +36,33 @@ def load_sentinel_data(
     Returns:
         xr.DataArray: A DataArray containing the 3 RGB bands from the visible spectrum
     """
+    dict_bands = {}
+    if len(band_names)==0:
+        return print('no bands selected')
+    for band in band_names :
+        dict_bands[band]=sorted(Path(sentinel_product_location).glob(f'GRANULE/*/IMG_DATA/R{resolution}m/*_{band}_*'))[0]
+    
+    list_dataarray =  []
 
-    ...  # TODO
-    raise NotImplementedError
+    for band_name, raster_path in dict_bands.items():
+        raster = open_rasterio(raster_path)
+        
+        list_dataarray.append(raster.assign_coords({'band': [band_name]}))
+
+    print(type(list_dataarray[0]))
+
+    rasters_concat = xr.concat(list_dataarray,dim = "band")
+    print(type(rasters_concat))
+    result = xr.where(rasters_concat==0,np.float32(np.NaN),(rasters_concat-1000)/10000)
```

Comment:
The literal values (`-1000` and `10000`) should be contained in variables 

See https://en.wikipedia.org/wiki/Magic_number_(programming) for why